### ISSUE TITLE
Reproducing results on the adult dataset

### ISSUE THREAD

**Issue by mortezamg63** [jsyoon0823/VIME#6]
I am trying to get improvement on VIME architecture for self-/semi-supervised learning. But it is not the case. I tried to check different latent space sizes. But I can not get to the results that are reported in the paper. At least, I expect to get better results compared to supervised learning. The table below shows my output. I have checked VIME on several other tabular datasets, I am getting the same outputs (no improvement on self-supervised and semi-supervised)

Training the adult dataset using original VIME
|Run | sup | self | semi |
|----|-----|------|-------|
|1|0.8359 |0.8311 |0.8251|
|2|0.8405 |0.8454 |0.8379|
|3|0.816  |0.8435 |0.8294|
|4|0.8361 |0.8376 |0.8309|

I used one-hot encoding for categorical data and the normalization for continuous features.  The following shows the columns that I use for training:

Age", "Workclass", "fnlwgt", "Education", "Martial Status",
            "Occupation", "Relationship", "Race", "Sex", "Capital Gain", "Capital Loss",
            "Hours per week", "Country", "Income"

Income is the target column. 

Could you please let me know what I can do to get improvement for self-supervised and semi-supervised learning.

**Comment by jsyoon0823** [comment]
Thank you for your comments.

I did not save the optimal hyper-parameters.
However, I remember that we need some extensive hyper-parameter tunings (not only hidden dimensions).
Definitely, I think you will get improved results with more extensive hyper-parameter tunings.

Thanks!

**Comment by somaonishi** [comment]
Are there any features that you have dropped from the dataset?

**Comment by mortezamg63** [comment]
"Age", "Workclass", "fnlwgt", "Education", "Martial Status",
"Occupation", "Relationship", "Race", "Sex", "Capital Gain", "Capital Loss",
"Hours per week", "Country", "Income"

Income is the target column.

The above features are the list of all features that I am using. I just drop "Education-Num" feature.
